### ISSUE TITLE
Owner unable to mute participant when tokens includes userId

### ISSUE THREAD

**Issue by swemail** [daily-co/react-native-daily-js#36]
Hello and thank you for this great service and library!

I found and issue muting participants in a meeting when the userId is included in the meeting token.

# Expected behavior

As a user joining with token including `owner: true` and `userId: "some-user-id"` I should be able to mute other participants joining with tokens including `owner: false` and `userId: "some-other-user-id"`

# Describe the bug (unexpected behavior)

When creating a token, via the api or self signed, and including the `userId` I get `update setAudio can't be applied to participant {{userId}}; skipping null` when as an owner calling `updateParticipant` on the daily call object like

```typescript
call.updateParticipant(participant.user_id, {setAudio: !muted})
```

If I exclude the `userId` from the token it works.

This is how we create tokens:

```typescript
import dayjs from 'dayjs';
import jwt from 'jsonwebtoken';
import {getDomainId} from './dailyApi';
import config from './config';

const {DAILY_API_KEY} = config;

export const generateSessionToken = async (
  roomName: string,
  isHost: boolean,
  expireDate: dayjs.Dayjs,
  userId?: string,
): Promise<string> => {
  const domainId = await getDomainId();

  const payload = {
    r: roomName,
    d: domainId,
    ud: userId,
    iat: dayjs().unix(),
    exp: expireDate.unix(),
    o: isHost,
  };

  return jwt.sign(payload, DAILY_API_KEY);
};
```

Is this expected behaviour? I could not find anything in the documentation about excluding `userId` from tokens for the owner to be able to mute others.

**Comment by filipi87** [comment]
Hi @swemail ,

We are glad that you are enjoying our API. =)

I was trying to replicate the issue, but I have not been able to reproduce it so far.

One thing that I have noticed from your error message is:
_**update setAudio can't be applied to participant {{user_id}}; skipping**_

The only way that I can get the same error message, is if I invoke the method with a `String` `'{{user_id}}'` instead of the actual participant id. Like this:
`callObject.updateParticipant('{{user_id}}', {setAudio: false})`

For instance, if I invoke with any other participant id which does not exist:
`callObject.updateParticipant('fakeId', {setAudio: false})`

I will receive:
`update setAudio can't be applied to participant fakeId; skipping`

So, can't you be facing some issue where you are not passing the right participant id which you wish to update ?

**Comment by swemail** [comment]
> Hi @swemail ,
> 
> We are glad that you are enjoying our API. =)
> 
> I was trying to replicate the issue, but I have not been able to reproduce it so far.
> 
> One thing that I have noticed from your error message is: _**update setAudio can't be applied to participant {{user_id}}; skipping**_
> 
> The only way that I can get the same error message, is if I invoke the method with a `String` `'{{user_id}}'` instead of the actual participant id. Like this: `callObject.updateParticipant('{{user_id}}', {setAudio: false})`
> 
> For instance, if I invoke with any other participant id which does not exist: `callObject.updateParticipant('fakeId', {setAudio: false})`
> 
> I will receive: `update setAudio can't be applied to participant fakeId; skipping`
> 
> So, can't you be facing some issue where you are not passing the right participant id which you wish to update ?

Ok, no, that was just me not wanting to expose a real id 😄, I get the error in the bottom with a real id. I will check again if I messed up witch id I'm sending

**Comment by filipi87** [comment]
Great. If you keep facing the same issue, please share what version of `react-native-daily-js` that you are using, so I can try to replicate the issue with the same version.

Also, another question, are you trying to invoke this method to also change the info from your local participant using this userId that you have created or only for the remote participants ?

**Comment by swemail** [comment]
> Great. If you keep facing the same issue, please share what version of `react-native-daily-js` that you are using, so I can try to replicate the issue with the same version.
> 
> Also, another question, are you trying to invoke this method to also change the info from your local participant using this userId that you have created or only for the remote participants ?

We're using `0.35.0`

I think I figured it out now. When I add `user_id` to the token, the `user_id` on the `participant` I get from the call object is that id that I set on the token. If I don't set `user_id` on the token `user_id` is the same as `session_id`. The documentation says exactly that. If I look at the type definition for `call.updateParticipant` it actually says that the first parameter is `sessionId` but we have sent `user_id` from the `participant` object and that was fine before since they where the same.   Maybe this explanation will help someone that find the same issue. Sorry for the confusion. Feel free to close this issue.
